network-device-mac.ks.in

```diff
@@ -30,7 +30,13 @@ shutdown
 # discoverable in sysfs yet so it picks its own name ksdevX and uses infname= to
 # name the device which results in binding also to HWADDR (because of ifname=
 # being used).
-check_device_config_value ksdev0 HWADDR __ANY ethernet mac-address __ANY
+if [ "@KSTEST_OS_NAME@" == "rhel" ] ; then
+    # RHEL-8-FAILURE mac binding for ifname= is not applied when updating connection,
+    # fixed upstream in commit a5cb6311e22587b539248bd5d2a227d08c0bb581
+    check_device_config_value ksdev0 HWADDR __NONE ethernet mac-address __NONE
```

Comment:
So that's a temporary hack, right? But this would fail on a "daily RHEL boot.iso" scenario again, as that would have the bug fix and thus behave normally again?

Reply:
This is not fixed in RHEL 8, and perhaps will not be as it is quite a corner case.